apps/web/components/sections/project/project-showcase.tsx

```
@@ -0,0 +1,55 @@
+	}
+
+	return (
+		<SectionContainer>
+			<SuccessGallery items={showcaseData} onMediaClick={openLightbox} />
+
+			{lightboxOpen && selectedMedia && (
+				<div className="fixed inset-0 bg-black bg-opacity-75 flex items-center justify-center z-50">
+					<button
+						onClick={closeLightbox}
+						className="absolute top-4 right-4 text-white text-2xl font-bold"
+					>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)